packages/quantic/force-app/main/default/lwc/exampleSearch/exampleSearch.html

```diff
@@ -1,42 +1,53 @@
-                        <c-quantic-tab label="Articles" expression="@sfkbid" engine-id={engineId}></c-quantic-tab>
-                        <c-quantic-tab label="Issues" expression="@jisourcetype AND NOT @jidocumenttype=&quot;WorkLog&quot;" engine-id={engineId}></c-quantic-tab>
-                        <c-quantic-tab label="Community" expression="@objecttype==&quot;Message&quot;" engine-id={engineId}></c-quantic-tab>
-                        <c-quantic-tab label="Files" expression="@boxdocumenttype==File OR @spcontenttype==Document" engine-id={engineId}></c-quantic-tab>
-                    </ul>
-                    <div class="slds-grid slds-wrap slds-grid_vertical-align-center slds-m-vertical_small">
-                        <c-quantic-summary class="slds-order_2 slds-medium-order_1 slds-size_1-of-1 slds-medium-size_7-of-12" engine-id={engineId}>
-                        </c-quantic-summary>
-                        <c-quantic-sort class="slds-order_1 slds-medium-order_2 slds-size_1-of-1 slds-medium-size_5-of-12" engine-id={engineId}>
-                        </c-quantic-sort>
-                    </div>
-                        <c-quantic-query-error engine-id={engineId}></c-quantic-query-error>
-                        <c-quantic-breadcrumb-manager engine-id={engineId}></c-quantic-breadcrumb-manager>
-                        <c-quantic-did-you-mean engine-id={engineId}></c-quantic-did-you-mean>
-                        <c-quantic-no-results engine-id={engineId}></c-quantic-no-results>  
-                        <c-quantic-result-list engine-id={engineId}></c-quantic-result-list>
-                        <div class="slds-var-m-vertical_medium">
-                            <c-quantic-pager engine-id={engineId}></c-quantic-pager>
-                            <c-quantic-results-per-page engine-id={engineId}></c-quantic-results-per-page>
-                        </div>
-                    </div>
+                <c-quantic-query-error engine-id={engineId}></c-quantic-query-error>
+                <c-quantic-breadcrumb-manager engine-id={engineId}></c-quantic-breadcrumb-manager>
+                <c-quantic-did-you-mean engine-id={engineId}></c-quantic-did-you-mean>
+                <c-quantic-no-results engine-id={engineId}></c-quantic-no-results>  
+                <c-quantic-result-list engine-id={engineId}></c-quantic-result-list>
+                <div class="slds-var-m-vertical_medium">
+                    <c-quantic-pager engine-id={engineId}></c-quantic-pager>
+                    <c-quantic-results-per-page engine-id={engineId}></c-quantic-results-per-page>
                 </div>
-            </c-quantic-search-interface>
+              </div>
+              <div class="slds-col slds-order_3 slds-large-order_3 slds-size_1-of-1 slds-large-size_3-of-12">
+                <c-quantic-recent-results-list engine-id={engineId} label="Recent Results"></c-quantic-recent-results-list>
```

Comment:
The `label` attribute could now be omitted here.

Reply:
sharp eyes as always. Like a hawk